boardgame-ui/src/locales/translations.csv

```diff
@@ -147,3 +147,22 @@ uservalidation.table.id,ID,
 uservalidation.table.username,Username,
 uservalidation.table.email,E-mail,
 uservalidation.table.validated,Validated ?,
+button.add-game,Add game,
+add-edit-game.title,Add game,
```

Comment:
Same comment as @FServais, maybe use 'Edit' instead of 'Add' since it is used for both creation and edition ?
#nitpicking

Reply:
You're totally right, I will split into two translations depending on case!